catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/Validator.java

```diff
@@ -66,14 +66,15 @@ void validateTypes(List<QName> types, String version) throws CswException {
             return;
         }
 
+        QName typeName = types.get(0);
+        QName cswOutputSchema = new QName(CswConstants.CSW_OUTPUT_SCHEMA, CswConstants.CSW_RECORD_LOCAL_NAME);
+        QName gmdOutputSchema = new QName(CswConstants.GMD_OUTPUT_SCHEMA, CswConstants.GMD_RECORD_LOCAL_NAME);
+
         if (types.size() == 1) {
-            if (!types.get(0)
-                    .equals(new QName(CswConstants.CSW_OUTPUT_SCHEMA,
-                            CswConstants.CSW_RECORD_LOCAL_NAME))) {
+            if (!typeName.equals(cswOutputSchema) && !typeName.equals(gmdOutputSchema)) {
                 throw createUnknownTypeException(types.get(0)
```

Comment:
I don't think that `toString()` is necessary for equality comparison here.  The `equals()` method for Qname compares the namespace and the local part